shuttle/driver_check_in/__init__.py

```diff
@@ -29,7 +33,13 @@ def load_driver_view(self):
         session['DRIVER-SELECT'] = json_data['view']
         if json_data['view'] == 'Location Check In':
             load = 'locations'
-            locations = self.shc.grab_locations()
+            locations = db.get_db_locations()
+            last_location = db.get_last_location()['location']
+            next_check_in = self.hc.grab_current_route()
+            next_location = next_check_in['location']
+            next_time = next_check_in['time']
+            if next_location == 'No more stops today' or next_location == 'No stops on the weekend':
+                next_location = 'North'
```

Comment:
I don't think there is a right or wrong answer but I believe you could just pass over the dictionary and use it in html instead of making all these additional variables since it might make it a little less cluttered. Up to you if you want to change it or not.